site/_posts/2023-03-30-digitalocean-oneclick-installation.md

```diff
@@ -0,0 +1,233 @@
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/), the Kubernetes command line interface.
+- [`helm`](https://helm.sh/docs/intro/install/), a helpful third-party tool for managing Kubernetes applications.
+
+Click the links above and follow the instructions for installing, configuring, and authenticating each tool for your system.
+
+## Installation
+
+You can install FusionAuth on DigitalOcean by clicking the "Install App" button at [this link](https://marketplace.digitalocean.com/apps/fusionauth).
+
+{% include _image.liquid src="/assets/img/blogs/digitalocean/digitalocean-install-app.png" alt="Install FusionAuth from DigitalOcean marketplace" class="img-fluid" figure=false %}
+
+On the next screen, select "Install". By default, this will install FusionAuth on a new Kubernetes cluster, but you can choose an existing cluster from the dropdown if you have one.  
+
+{% include _image.liquid src="/assets/img/blogs/digitalocean/digitalocean-select-cluster.png" alt="Select Kubernetes cluster to install FusionAuth on" class="img-fluid" figure=false %}
+
+You will be taken to the cluster configuration page. The default values here will work. You may want to choose the project to install under and give the cluster a more readable name. When you're done, scroll down to the bottom of the page and click "Create Cluster". Three nodes will be created so that FusionAuth works properly: one for the database, one for Elasticsearch, and one for the FusionAuth API.
+
+{% include _image.liquid src="/assets/img/blogs/digitalocean/digitalocean-create-cluster.png" alt="Settings configuration for new Kubernetes cluster" class="img-fluid" figure=false %}
+
+FusionAuth may take several minutes to install. Once installed, click on the cluster name to access guides, resources, settings, and more. Select the "Overview" panel and click "Get Started". Navigate to the second step, "Connecting to Kubernetes". There you will find a command on the "Automated" tab with the appropriate value for `<YOUR_CLUSTER_ID>`.
+
+{% include _image.liquid src="/assets/img/blogs/digitalocean/digitalocean-connecting-to-kubernetes.png" alt="command line prompt to connect to your Kubernetes cluster" class="img-fluid" figure=false %}
+
+If you have already installed the [DigitalOcean command line tool](https://docs.digitalocean.com/reference/doctl/how-to/install/), run the following command in your terminal:
+
+```sh
+doctl kubernetes cluster kubeconfig save <YOUR_CLUSTER_ID>
+```
+
+This command should return the following:
+
+```sh
+Notice: Adding cluster credentials to kubeconfig file found in "/Users/<USERNAME>/.kube/config"
+Notice: Setting current-context to <YOUR_CLUSTER_NAME>
+```
+
+FusionAuth is now running on a publicly accessible IP address. To retrieve this address, run the following command in your terminal:
+
+```sh
+export SERVICE_IP=$(kubectl get svc --namespace fusionauth fusionauth -o jsonpath='{.status.loadBalancer.ingress[0].ip}') \
```

Comment:
```suggestion
export SERVICE_IP=$(kubectl get svc --namespace fusionauth fusionauth -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Remove backslash at the end of the first line (consistent with other places this snippet occurs).  Or insert a semicolon before the backslash `; \`